how-it-works/2_chainkeytech/04_chain_key_tokens.card.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Chain-key tokens
+abstract: This feature brings a Bitcoin analogue to ICP can be transferred at the speed of the ICP and with low transaction fees.
```

Comment:
As discussed, maybe "Bitcoin analogue" is not the best terminology for an abstract.
The alternative would be something like "a token that is backed 1:1 by bitcoin".

Reply:
Moreover, ICP -> Internet Computer